framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -3106,7 +3106,7 @@ public boolean validateTypeOf(Tree tree) {
             }
             break;
         default:
-            type = atypeFactory.getAnnotatedType(tree);
+        	type = atypeFactory.getDefaultedAnnotatedType(tree);
```

Comment:
Please don't make gratuitious whitespace changes.  Source code files should never contain tab characters.

Can you please provide a test case that fails before this change and passes after the change?  That will motivate the need for the change.